website/source/docs/configuration/storage/spanner.html.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: "docs"
+page_title: "Google Cloud Spanner - Storage Backends - Configuration"
+sidebar_current: "docs-configuration-storage-spanner"
+description: |-
+  The Google Cloud Spanner storage backend is used to persist Vault's data in
+  Spanner, a fully managed, mission-critical, relational database service that
+  offers transactional consistency at global scale.
+---
+
+# Spanner Storage Backend
+
+The Google Cloud Spanner storage backend is used to persist Vault's data in
+[Spanner][spanner-docs], a fully managed, mission-critical, relational database
+service that offers transactional consistency at global scale, schemas, SQL, and
+automatic, synchronous replication for high availability.
+
+- **High Availability** – the Google Cloud Spanner storage backend supports high
+  availability. Because the Google Cloud Spanner storage backend uses the system
+  time on the Vault node to acquire sessions, clock skew across Vault servers
+  can cause lock contention.
```

Comment:
what is a user supposed to do if there is lock contention?

Reply:
Cry

Just kidding. Basically they need to resolve the skew using something like ntp and restarting services. None of the other backends documented how to fix skew, and, since it's not specific to Spanner, I don't know if it belongs here. What do you think?

Reply:
Most (all?) other HA backends aren't time-specific, they're session-specific, so skew isn't really an issue.

Reply:
DynamoDB is 😄

Reply:
I think there is a way to do an exclusive lock-for-writing on a document. I don't know what the failure conditions are. It's possible they just don't account for clock skew at all.